plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -306,7 +306,6 @@
 import static io.trino.plugin.hive.util.HiveWriteUtils.createPartitionValues;
```

Comment:
Can we reword the commit message as `Create an external table location for Hive if it does not exist.`